### PR TITLE
fix: add v3 for coo template

### PR DESCRIPTION
**Pull request by ChenRen93**

### CODE REVIEW

File: src/templates/ChaftaCoo/ChaftaCooTemplate.tsx [hunk 1]
```diff
@@ -55,7 +56,8 @@ export const ExporterSection: FunctionComponent<TemplateProps<ChaftaCooDocument>
   handleObfuscation,
   isPrivacyOn,
 }) => {
-  const exporter = document.supplyChainConsignment?.exporter;
+  const documentData = utils.isRawV3Document(document) ? (document.credentialSubject as ChaftaCooDocument) : document;
+  const exporter = documentData.supplyChainConsignment?.exporter;
```
Comment (simboonlong) [inline src/templates/ChaftaCoo/ChaftaCooTemplate.tsx:60]: wanna util at oa side?

File: application/app.tsx [hunk 2]
```diff
@@ -138,11 +138,10 @@ export const App: React.FunctionComponent<AppProps> = ({ documents }): React.Rea
           </div>
           {documents.length === 0 && <div>Please configure the application and provide at least one document</div>}
           {documents.map((d, index) => {
-            console.log("D", d.document.$template.name);
             return (
               <div
                 key={index}
-                data-testid={d.document.$template.name || ""}
+                data-testid={d.document.$template?.name || d.document.openAttestationMetadata?.template.name || ""}
                 className={`document ${document === d ? "active" : ""}`}
```
Comment (simboonlong) [inline application/app.tsx:145]: ah okay then if that's the case lets keep to this instead?
```suggestion
                data-testid={`document-${index}`}
                className={`document ${document === d ? "active" : ""}`}
```

then the tests part needa update

File: src/templates/ChaftaCoo/ChaftaCooTemplate.tsx [hunk 3]
```diff
@@ -354,8 +364,9 @@ export const CertificationSection: FunctionComponent<TemplateProps<ChaftaCooDocu
 export const ChaftaCooTemplate: FunctionComponent<TemplateProps<ChaftaCooDocument> & HasPrivacyToggle> = (props) => {
   const [isPrivacyOn, setIsPrivacyOn] = useState(false);
```
Comment (simboonlong) [inline src/templates/ChaftaCoo/ChaftaCooTemplate.tsx:365]: want to pass in the specific props into the sections at this level? rather then repeating the get `documentData`

File: application/app.tsx (hunk 2 above)
Comment (ChenRen93) [reply application/app.tsx]: okay, done

File: src/templates/ChaftaCoo/ChaftaCooTemplate.tsx (hunk 3 above)
Comment (ChenRen93) [reply src/templates/ChaftaCoo/ChaftaCooTemplate.tsx]: ah okay, done.

**Review by simboonlong — APPROVED** [review]
lgtm

**Pull request merged**